src/client/auth/actions.js

```diff
@@ -33,14 +33,23 @@ function validateForm(fields) {
 }
 
 function validateCredentials(fields) {
-  // Simulate lengthy async action.
   return new Promise((resolve, reject) => {
-    setTimeout(() => {
-      if (fields.password === 'pass1')
+
+    var xhr = new XMLHttpRequest();
```

Comment:
Well, as the Este.js acts as a `next-gen` `to-go` `everything-but-the-kitchen-sink` boilerplate, that'd be a good solution to actually use it along with polyfill. On the other hand, this method acts as a super cruel example demonstrating the general purpose so not sure if there's a need to bundle such a dependency. But as we are planning to add more fetch requests, using something `native` yet simpler than XHR would be a good choice.
@steida ?